src/renderer/coremods/commands/commands.ts

```diff
@@ -0,0 +1,382 @@
+      } catch (err) {
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0xdd2d2d,
+              title: "Error",
+              description: err as string,
+            },
+          ],
+        };
+      }
+    },
+  },
+  {
+    name: "list",
+    description: "List all plugins/themes",
+    usage: '/list <true/false> <"plugin"/"theme"> <>',
+    options: [
+      {
+        name: "send",
+        description: "Whether you want to send this or not.",
+        type: ApplicationCommandOptionType.Boolean,
+        required: false,
+      },
+      {
+        name: "addon type",
+        description: "Choose what type of addons to list.",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        choices: [
+          {
+            name: "Theme",
+            displayName: `List themes`,
+            value: "theme",
+          },
+          {
+            name: "Plugin",
+            displayName: `List plugins`,
+            value: "plugin",
+          },
+        ],
+      },
+      {
+        name: "version",
+        description: "Whether you want to add version info.",
+        type: ApplicationCommandOptionType.Boolean,
+        required: true,
+      },
+      {
+        name: "type",
+        description: "Whether you want to send either only enabled, disabled or all themes.",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        choices: [
+          {
+            name: "Enabled",
+            displayName: "Enabled",
+            value: "enabled",
+          },
+          {
+            name: "Disabled",
+            displayName: "Disabled",
+            value: "disabled",
+          },
+          {
+            name: "Both",
+            displayName: "Both",
+            value: "default",
+          },
+        ],
+      },
+    ],
+    executor: (options) => {
+      try {
+        const send = (options.find((o) => o.name === "send")?.value as boolean) ?? false;
+        const addonType = options.find((o) => o.name === "addon type")?.value;
+        const version = options.find((o) => o.name === "version")?.value;
+        const listType = options.find((o) => o.name === "type")?.value;
+        switch (addonType) {
+          case "plugin": {
+            const allPlugins = Array.from(plugins.plugins.values())
+              .map((p) => p.manifest)
+              .sort((a, b) => a.name.localeCompare(b.name));
+            const enablePlugins = allPlugins?.filter((p) => !plugins.getDisabled().includes(p.id));
+            const disabledPlugins = allPlugins?.filter((p) => plugins.getDisabled().includes(p.id));
+            const enabledString = `• ${enablePlugins
+              .map((p) => (version ? `${p.name} (${p.version})` : p.name))
+              .join("\n• ")}`;
+            const disabledString = `• ${disabledPlugins
+              .map((p) => (version ? `${p.name} (${p.version})` : p.name))
```

Comment:
`v`-prefix the version number like in settings

Reply:
`(v${p.version})`

like so?